src/Jackett.Common/Indexers/AnimeBytes.cs

```diff
@@ -345,15 +345,10 @@ private async Task<IEnumerable<ReleaseInfo>> GetResults(TorznabQuery query, stri
                                 releasegroup = string.Empty;
                             }
 
-                            var infoString = "";
-
-                            for (int i = 0; i + 1 < releaseTags.Count(); i++)
-                            {
-                                if (releaseTags[i] == "Raw" && !AllowRaws)
-                                    continue;
-                                infoString += "[" + releaseTags[i] + "]";
-                            }
+                            if (!releaseTags.All(tag => (tag.ToLowerInvariant() != "raw")) && !AllowRaws)
```

Comment:
Maybe use `.Contains` instead for better readability?

`if (releaseTags.Contains("raw", StringComparer.InvariantCultureIgnoreCase) && !AllowRaws)`